src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  count: 0,
+};
+
+export const catagoriySlice = createSlice({
+  name: 'catagory',
+  initialState,
+  reducers: {
+    status: (state) => {
+      state.count = 'under construction';
+    },
+  },
+});
```

Comment:
- [ ] Kindly apply the recommendation to this file too to fit in the requirement here.

Reply:
Hi, @Olamarx thank you for your constructive comments.